lib/v2/oreno3d/router.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function (router) {
+    router.get('/search/:sort?/:keyword', require('./search'));
+    router.get('/characters/:sort?/:characterid', require('./characters'));
+    router.get('/authors/:sort?/:authorid', require('./authors'));
+    router.get('/tags/:sort?/:tagid', require('./tags'));
+    router.get('/origins/:sort?/:originid', require('./origins'));
```

Comment:
Please sort route paths in alphabetical order
```suggestion
    router.get('/authors/:sort?/:authorid', require('./authors'));
    router.get('/characters/:sort?/:characterid', require('./characters'));
    router.get('/origins/:sort?/:originid', require('./origins'));
    router.get('/search/:sort?/:keyword', require('./search'));
    router.get('/tags/:sort?/:tagid', require('./tags'));
```